.travis.yml

```diff
@@ -22,6 +22,7 @@ before_script:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sleep 3; fi
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then npm install; fi
 script:
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then npm run compile; fi
```

Comment:
what does compile do?  Why are we adding it now?

Reply:
@mydiemho this checks to make sure it will compile with typescript. it should have been added from the start

Reply:
@mydiemho It basically runs the typescript compiler.  We have see cases where lint and tests pass but there is a minor TS issue that prevents the electron app from launching.  This will run the TS compiler `tsc` in addition to linting to verify that this doesn't happen.